includes/class-wc-stripe-webhook-handler.php

```
@@ -1781,6 +1790,74 @@ private function get_payment_method_type_from_charge( $charge ) {
+	/**
+	 * Handle the Agentic Checkout customization hook.
+	 *
+	 * This parameter is expected to generate both an HTTP status code and a JSON response.
+	 *
+	 * @since 10.5.0
+	 * @param stdClass $event The webhook event from Stripe.
+	 * @return void
+	 */
+	private function process_agentic_customization_hook( stdClass $event ): void {
+		try {
+			$event               = new WC_Stripe_Agentic_Customize_Checkout_Event( $event );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '1833', column '31'). (undefined)

(MissingImport)